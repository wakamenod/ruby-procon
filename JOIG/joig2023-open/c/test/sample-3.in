10 10 10000
589 2398 6567 28817 29177 31636 45468 66751 82282 97509
2196 54498 80474 61644 18007 38759 85590 72172 79533 69959
