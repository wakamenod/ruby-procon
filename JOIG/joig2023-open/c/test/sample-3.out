9798
970
8192
4893
0
3291
6692
4579
7251
6792
